src/core/workspace/folder-to-project.ts

```diff
@@ -147,14 +168,18 @@ function _processResource(key: ResourceKey, buffer: Buffer, destFolder: string)
         ? SimDataResource.from(buffer)
         : SimDataResource.fromXml(buffer);
 
-      const xmlContent = simdata.toXmlDocument().toXml();
-
-      // TODO: insert group and instance override if tuning not found
+      var dest: string;
+      const val = instanceMap.get(key.instance);
+      if (val) {
+        simdata.instance.name = val[0] + "_SimData";
```

Comment:
I see why you're doing this, and agree that it's a nice QOL feature, but I do not want the package > folder conversion to alter files at all – it should retain the exact content of the original files.